src/utils/collection.ts

```diff
@@ -110,7 +99,7 @@ export function getCollectionSize(fileList: FileList | File[]): number {
   let sum = 0
 
   for (let i = 0; i < fileList.length; i++) {
-    const file = fileList[i] as WebkitFile
+    const file = fileList[i] as File
```

Comment:
```suggestion
    const file = fileList[i]
```
The type cast is no longer needed in 4.5.4